mbuild/conversion.py

```diff
@@ -83,7 +83,7 @@ def load(
     ignore_box_warn : bool, optional, default=False
         If True, ignore warning if no box is present. Defaults to True when
         loading from SMILES
-    **kwargs : keyword arguments
+    ``**kwargs`` : keyword arguments
```

Comment:
why do we need the backticks here?

Reply:
Sphinx will not build without errors otherwise